### ISSUE TITLE
Install fails with multiple devices

### ISSUE THREAD

**Issue by NGPixel** [Simizfo/WSATools#34]
On my machine, running `adb devices` returns 2 devices:

```sh
PS C:\> adb devices
List of devices attached
emulator-5554   device
localhost:58526 device
```

This results in WSATools failing with error `adb.exe: more than one device/emulator` when trying to install an APK.

The app should check whether multiple devices are listed first and if so, either automagically select the WSA one or let the user select the correct one, then pass the device serial in the install command in order to prevent this error, e.g.:

```sh
adb -s 192.168.0.100:12345 install com.some.app.apk
```

(I did try to somehow turn off this emulator entry but it always come back or fail with some other error.)

**Comment by Simizfo** [comment]
Thanks for reporting. This definitely needs to be implemented.

**Comment by Simizfo** [comment]
Closed #35 as it was a duplicate. Also got another report on Direct Messages on Telegram.

Looks like this might get priority and be solved ASAP then

**Comment by keatit71** [comment]
I'm not getting an error from having multiple devices but any apk I try to install with wsatools just sits there installing infinitely and i thought it might be the same issue since I have multiple devices listed as well. Installing with another app already installed like apkpure works fine.

**Comment by StefanHiersekorn** [comment]
I've got the same issue. Didn't find a way to resolve it yet :-/

**Comment by Riolu69** [comment]
Just open adb folder > right click and open terminal > write "adb kill-server" and all be work.

**Comment by NGPixel** [comment]
> Just open adb folder > right click and open terminal > write "adb kill-server" and all be work.

No it doesn't.

**Comment by Simizfo** [comment]
Took me a while, but I'm finally done with university and I'm ready to pick this project back up.

Please accept my excuses for this long downtime.

This should be fixed in 0.2.3, I've just uploaded it to the Store, will come in a few days! Let me know if it works and I'll close this.

**Comment by Simizfo** [comment]
Closing this as it's fixed.
